packages/aeproject-cli/aeproject-init/init.js

```diff
@@ -59,8 +62,44 @@ const createAEprojectProjectStructure = async (shape) => {
     print('===== AEproject was successfully initialized! =====');
 }
 
+const compareSdkVersions = async (_sdkVersion, cwd) => {
+    // get current sdk version
+    let userPackageJson = JSON.parse(fs.readFileSync(path.join(cwd, './package.json'), 'utf8'));
+    let userSdkVersion = userPackageJson.dependencies['@aeternity/aepp-sdk'];
+
+    if (userSdkVersion) {
+        let userVersioning = userSdkVersion.split('.');
+        let updateToVersioning = _sdkVersion.split('.');
+
+        let promptMessage = `Found newer or different version of sdk ${ userSdkVersion }. Keep it, instead of ${ _sdkVersion }?`;
+
+        for (let i = 0; i < 3; i++) {
+            let user = userVersioning[i];
+            let updateTo = updateToVersioning[i];
+
+            if (!isNaN(updateTo) && !isNaN(user)) {
+                if (parseInt(user) > parseInt(updateTo)) {
+                    if (await promptUpdate(promptMessage)) {
+                        _sdkVersion = userSdkVersion;
+                        break;
+                    }
+                }
+            } else if (!isNaN(updateTo) && isNaN(user)) {
```

Comment:
why do you need the **user** to be **nan** here? why in the first place you need the **if** here?

Reply:
if user's version is '5.0.0rc4', '0rc4' can not be parsed and assume that its version is higher

Reply:
Does the npm packages have, **rc** versions?

Reply:
no, but if it is a github link/repo '"package-name": "username/repo#branch-name"'

Reply:
valid point. In my opinion you can remove the **!isNaN(updateTo)** from both **if** statements and then change the **else if** to only else